packages/redux-requests-graphql/src/graphql-driver.js

```diff
@@ -4,9 +4,10 @@ import { extractFiles } from 'extract-files';
 const calculateProgress = progressEvent =>
   parseInt((progressEvent.loaded / progressEvent.total) * 100);
 
-export const createDriver = ({ url }) => {
+export const createDriver = ({ url, ...rest }) => {
   const axiosInstance = axios.create({
     baseURL: url,
+    ...rest
   });
 
   return (requestConfig, requestAction, driverActions) => {
```

Comment:
Thanks for your research and apologies for late reply! Could we please use https://www.typescriptlang.org/docs/handbook/utility-types.html#picktype-keys instead of `omit`? So we will be strict and new axios types won't be added without our knowledge.

Rgarding attrs I would actually support are:
- headers
- timeout
- auth
- xsrfCookieName
- xsrfHeaderName
- proxy

I think only those would be handy for graphql driver, what do you think?

>Then any of the options can still be configured per request with axiosOptions

That would be excellent!